client/state/imports/site-importer/actions.js

```diff
@@ -130,9 +130,15 @@ export const importSite = ( {
 	dispatch( recordTracksEvent( 'calypso_site_importer_start_import_request', trackingParams ) );
 	dispatch( startSiteImporterImport() );
 
-	wpcom
-		.undocumented()
-		.importWithSiteImporter( siteId, toApi( importerStatus ), params, targetSiteUrl )
+	wpcom.req
+		.post( {
+			path: `/sites/${ siteId }/site-importer/import-site?${ stringify( params ) }`,
```

Comment:
Can we avoid `stringify` here and pass a `query: params` param?

Reply:
Good call 👍

Reply:
Nope, `query: params` doesn't do anything. Likely because we have `formData` and are handling it as a `content-type: multipart/form-data` request.

I say, let's keep it as-is. `stringify` is already used in this file anyway.

Reply:
Oh, that's another silly behavior of `wpcom.req.post`: if you call the `.post( params, query, body, fn )` without the `query` arg, it will be defaulted to `{}` even when there's already `params.query` -- that will be overwritten 🤦‍♂️